nmdc_runtime/api/v1/workflows/activities.py

```diff
@@ -46,10 +46,6 @@ async def post_activity(
     """
     _ = site  # must be authenticated
     try:
-        # verify activities in activity_set are nmdc-schema compliant
```

Comment:
Do we still want to perform this check for certain kinds of activtities rather than dropping entirely